Source/WebCore/platform/graphics/texmap/BitmapTexture.cpp

```diff
@@ -209,6 +216,11 @@ void BitmapTexture::updateContents(NativeImage* frameImage, const IntRect& targe
     int bytesPerLine = cairo_image_surface_get_stride(surface);
 
     updateContents(imageData, targetRect, offset, bytesPerLine, PixelFormat::BGRA8);
+#elif USE(SKIA)
+    sk_sp<SkImage> surface = frameImage->platformImage();
+    SkPixmap pixmap;
+    if (surface->peekPixels(&pixmap))
+        updateContents(pixmap.addr(), targetRect, offset, pixmap.rowBytes(), PixelFormat::BGRA8);
```

Comment:
Is this SkImage always unaccelerated?

Reply:
Yes, at the moment.
https://github.com/WebKit/WebKit/blob/1b0ca739c27ecccb811e64e46232f33b1401c5ca/Source/WebCore/platform/graphics/texmap/BitmapTexture.cpp#L221-L223
`ImageBufferOptions::Accelerated` has to be specified.